core/modules/script.py

```diff
@@ -45,18 +82,33 @@ class ScriptNodeSkel(BaseScriptAbstractSkel):
         defaultValue=False,
     )
 
-    plugin = BooleanBone(
-        descr="Is plugin?",
-        defaultValue=False
-    )
-
     name = StringBone(
         descr="Folder",
         required=True,
         vfunc=lambda value: not DIRECTORY_PATTERN.match(value)
     )
 
 
+class ButtonbarSkel(skeleton.RelSkel):
```

Comment:
In terms of the vi, we normally talk about actions, and not "buttonbars".
```suggestion
class ScriptLeafActionSkel(skeleton.RelSkel):
```